build.sbt

```
@@ -286,16 +280,12 @@ lazy val profile = project
   )
 
 lazy val docs = project
-  .in(file("docs"))
-  .enablePlugins(SitePreviewPlugin, ParadoxSitePlugin)
+  .in(file("modules/docs"))
+  .enablePlugins(NoPublishPlugin, AutomateHeaderPlugin)
+  .enablePlugins(TypelevelSitePlugin)
```

It already enables the no-publish plugin.
```suggestion
  .enablePlugins(TypelevelSitePlugin, AutomateHeaderPlugin)
```